packages/devtools_app/lib/src/ui/analytics.dart

```diff
@@ -342,26 +358,26 @@ Future<bool> get isSurveyActionTaken async {
 }
 
 /// Set DevTools property value 'surveyActionTaken' stored in the file
-/// '~\.devtools'.
+/// '~\.devtools'.  If activeSurvey is set the activeSurvey actionTaken is set
+/// otherwise original actionTaken is set (first survey).
```

Comment:
I thought we decided to throw errors instead of doing anything with the original survey values?